### PR TITLE
gst-python 1.4.0 (new formula)

### PR DESCRIPTION
**Pull request by hyunkim9123**
This is to resolve the issue at https://github.com/Homebrew/homebrew/issues/32381

### CODE REVIEW

**Comment by hyunkim9123** [comment]
Looks I need a little modification. Will upload new patch soon.

File: Library/Formula/gst-python.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+  test do
+    system "gst-inspect-1.0", "--vd", "python"
```
Comment (mikemcquaid) [inline Library/Formula/gst-python.rb]: This is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/24791/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_gst_python/

File: Library/Formula/gst-python.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class GstPython < Formula
+  homepage "http://gstreamer.freedesktop.org/modules/gst-python.html"
+  url "http://gstreamer.freedesktop.org/src/gst-python/gst-python-1.4.0.tar.xz"
+  version "1.4.0"
+  sha256 "b1e40c29ceb41b03f08d38aca6056054f0341d0706276326dceeec6ac8d53d3e"
+
+  head do
```
Comment (mikemcquaid) [inline Library/Formula/gst-python.rb]: Just use `head "http:/"` etc.

**Comment by mikemcquaid** [comment]
Is this available in PyPi?

**Comment by hyunkim9123** [comment]
@mikemcquaid Thank you for comments. Uploaded new pat with built correctly. I did not see this in PyPi.

File: Library/Formula/gst-python.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+class GstPython < Formula
+  homepage "http://gstreamer.freedesktop.org/modules/gst-python.html"
+  url "http://gstreamer.freedesktop.org/src/gst-python/gst-python-1.4.0.tar.xz"
+  sha256 "b1e40c29ceb41b03f08d38aca6056054f0341d0706276326dceeec6ac8d53d3e"
+
+  head "http:/"
```
Comment (mikemcquaid) [inline Library/Formula/gst-python.rb]: This is wrong.

File: Library/Formula/gst-python.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  depends_on "pygobject3"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+  test do
```
Comment (mikemcquaid) [inline Library/Formula/gst-python.rb]: newline above this line

File: Library/Formula/gst-python.rb (hunk 3 above)
Comment (hyunkim9123) [inline Library/Formula/gst-python.rb]: Is the new commit what you expected? http://gstreamer.freedesktop.org/src/gst-python/gst-python-1.4.0.tar.xz

File: Library/Formula/gst-python.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
+class GstPython < Formula
+  homepage "http://gstreamer.freedesktop.org/modules/gst-python.html"
+  url "http://gstreamer.freedesktop.org/src/gst-python/gst-python-1.4.0.tar.xz"
+  sha256 "b1e40c29ceb41b03f08d38aca6056054f0341d0706276326dceeec6ac8d53d3e"
+
+  head "http://gstreamer.freedesktop.org/src/gst-python/gst-python-1.4.0.tar.xz"
```
Comment (mikemcquaid) [inline Library/Formula/gst-python.rb]: Just remove this line, thanks.

**Comment by hyunkim9123** [comment]
@mikemcquaid  The most recent patch passes the testing and looks good. Please review.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**